src/commands/remove-demo.ts

```diff
@@ -78,22 +99,21 @@ module.exports = {
         }
       })
 
-    // Recurisvely remove any empty directories in TARGET_DIR
-    const removeEmptyDirs = (dir: string) => {
-      const files = filesystem.list(dir).filter((file) => file.includes("node_modules") === false)
-      if (files.length === 0) {
-        filesystem.remove(dir)
-        p(`Removed empty directory '${dir}'`)
-        return
-      }
-      files.forEach((file) => {
-        const filePath = `${dir}/${file}`
-        if (filesystem.isDirectory(filePath)) {
-          removeEmptyDirs(filePath)
-        }
+    const emptyDirPaths = filesystem
+      .cwd(TARGET_DIR)
+      .find({
+        matching: MATCHING_GLOBS,
+        recursive: true,
+        files: false,
+        directories: true,
       })
-    }
-    removeEmptyDirs(TARGET_DIR)
+      .map((path) => pathlib.join(TARGET_DIR, path))
+      .filter((path) => !filesystem.list(path)?.length)
```

Comment:
we can do the same here. we don't want to delete all empty dirs, only those that match